core-common/src/main/java/org/mapstruct/Mapper.java

```diff
@@ -112,4 +112,15 @@
      * @return The strategy to be applied when {@code null} is passed as source value to the methods of this mapper.
      */
     NullValueMappingStrategy nullValueMappingStrategy() default NullValueMappingStrategy.DEFAULT;
+
+    /**
+     * The strategy to use for applying {@code @Mapping} configurations of prototype methods in the interface specified
```

Comment:
I'm wondering, is it restricted to bean mapping methods actually or also iterable/map mapping methods? The usefulness is much higher for the former, but I'm wondering whether it might make sense to allow to inherit configurations for the other types, too, at some point?